nifi-assembly/pom.xml

```diff
@@ -906,11 +918,6 @@ language governing permissions and limitations under the License. -->
             <version>2.0.0-SNAPSHOT</version>
             <type>nar</type>
         </dependency>
-        <!-- AspectJ library needed by the Java Agent used for native library loading (see bootstrap.conf) -->
```

Comment:
@exceptionfactory I think we need to keep the aspectj references here and all others below as that is how our native lib loading mechanism works and that is a powerful/key feature for any nars that might need to load native libs and still benefit from classloader isolation even in the event of native libs.

Reply:
Thanks for the feedback. Although loading native libraries may not be necessary for most libraries, this makes sense for the edge cases, and support for the aspect implementation also available. The latest commit restores the AspectJ references.